lib/api_auth/header_compare.ex

```diff
@@ -2,20 +2,20 @@ defmodule ApiAuth.HeaderCompare do
   @moduledoc false
 
   alias ApiAuth.Utils
+  alias Plug.Crypto
 
   def wrap(valid, request) do
     {:ok, valid, request}
   end
 
   def compare(hc, keys) do
-    compare(hc, keys, &SecureCompare.compare/2)
+    compare(hc, keys, &Crypto.secure_compare/2)
   end
```

Comment:
My understanding here: where previously, we were passing in a pointer to &SecureCompare.compare, we've replaced that with &Crypto.secure_compare.

The new lib doesn't require any kind of config args?

Reply:
https://elixir-lang.org/getting-started/modules-and-functions.html#function-capturing < here's some more info on the `& /2` syntax. both methods have arity of 2 and accept 2 binaries no changes should be needed (but we'll make sure this still works on non-prod)